src/background/adblocker.js

```diff
@@ -92,21 +92,28 @@ async function updateEngines() {
 
     if (enabledEngines.length) {
       let updated = false;
+
       // Update engines from the list of enabled engines
-      for (const id of enabledEngines) {
-        if (id === engines.CUSTOM_ENGINE) continue;
-        updated = (await engines.update(id).catch(() => false)) || updated;
-      }
+      await Promise.all(
+        enabledEngines.map((id) =>
+          engines.update(id).then(
+            (v) => {
+              updated = updated || v;
+            },
+            () => {},
+          ),
+        ),
+      );
 
       // Reload the main engine after all engines are updated
       if (updated) await reloadMainEngine();
 
+      // Update timestamp after the engines are updated
+      await store.set(Options, { filtersUpdatedAt: Date.now() });
+
       // Update TrackerDB engine
       trackerdb.setup.pending && (await trackerdb.setup.pending);
       await engines.update(engines.TRACKERDB_ENGINE).catch(() => null);
-
-      // Update timestamp after the engines are updated
-      await store.set(Options, { filtersUpdatedAt: Date.now() });
```

Comment:
With this we no longer know if trackerdb was updated

Reply:
I did not think about it in this way. You're right.